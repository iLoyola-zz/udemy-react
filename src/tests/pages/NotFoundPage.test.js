import React from 'react';
import { shallow } from 'enzyme';

import NotFoundPage from '../../pages/NotFoundPage';

test( 'Should render the Not Found Page', () => {
	const wrapper = shallow( <NotFoundPage /> );
	expect( wrapper ).toMatchSnapshot();
});