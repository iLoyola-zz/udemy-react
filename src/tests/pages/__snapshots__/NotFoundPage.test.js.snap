// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render the Not Found Page 1`] = `
<div>
  404! Not Found Page!
  <Link
    replace={false}
    to="/"
  >
    Go Home
  </Link>
</div>
`;
